### DISCUSSION TITLE
How do I donate?

### DISCUSSION THREAD

**Discussion by AnsgarTOdinson** [Open-Shell/Open-Shell-Menu#747] [General]
This program is the one thing that keeps me from hating having to use my computer.  I am hoping to see it continue to be updated and developed as that new awful looking windows 11 makes its rollout.

Where can I donate to this project to make sure the wonderful people making our lives so much easier are compensated for all their hard work?

**Comment by bonzibudd** [comment]
I don't see any donations set up, but you can try to ping or contact the maintainers individually: https://github.com/orgs/Open-Shell/people

**Comment by DigitalSorceress** [comment]
I would love to donate something - even if they just want to set up a "buy me a coffee" 

I genuinely don't know if I could keep using Windows 10 and up without this - I use StartAllBack / StartIsBack to fix some things OpenShell does not but I prefer the OpenShell menu (I use the classic  - windows 2000 like menu as I HATE even win7 etc...)

Seriously I want to see this continue and would put my money where my lack of c++ is

**Comment by karaldo** [comment]
Me too. I'll be glad to make a donation so that this Open Shell project can continue developing.
